scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -2274,14 +2275,14 @@ EOF
       fi
     fi
     echo
-    
+
     owner_wallet="${wallet_name}"
     getBaseAddress ${owner_wallet}
     getBalance ${base_addr}
 
     if [[ ${lovelace} -gt 0 ]]; then
       if [[ -n ${wallet_count} && ${wallet_count} -gt ${WALLET_SELECTION_FILTER_LIMIT} ]]; then
-        say "$(printf "%s\t${FG_CYAN}%s${NC} ADA" "Funds in base address for owner wallet:"  "$(formatLovelace ${lovelace})")" "log"
+        say "$(printf "%s\t${FG_CYAN}%s${NC} ADA" "Funds in base address + rewards for owner wallet:"  "$(formatLovelace `expr ${lovelace} + ${reward_lovelace}`)")" "log"
```

Comment:
$(( lovelace + reward_lovelace ))

Reply:
Thanks. I've familiar with multiple methods to add integers, but which is new/old and benefits of each I don't keep committed to memory, which I should :+1: